### ISSUE TITLE
Run our own Ethereum testnet for e2e tests

### ISSUE THREAD

**Issue by nearmax** [near/rainbow-bridge#292]
Ganache does not provide PoW testing, and Ropsten and other Ethereum testnets need real tokens that we would consume really fast with heavy tests running on Nayduck. We need to create our own Ethereum testnet and use it for tests like these.

**Comment by chefsale** [comment]
Currently we have a PoA network near-eth on single node on GCP, most likely we can change the config to be PoW cc: @mhalambek

**Comment by djsatok** [comment]
@chefsale , is PoA Ethereum network still there now?

**Comment by chefsale** [comment]
Nope, we spinned it down, as it wasn't needed. Only the ethereum nodes on rinkeby and ropsten. I believe @mhalambek has still the terraform configuration,so could be spinned up if needed.

**Comment by djsatok** [comment]
@chefsale , @mhalambek , no need to do it right now. We're assessing the priorities.
